serverlessworkflow/modules/ROOT/pages/cloud/operator/build-and-deploy-workflows.adoc

```diff
@@ -57,55 +163,164 @@ Double-check your `$\{HOME\}/.docker/config.json`. If you're using local desktop
 
 === Configure the {operator_name} (i.e. registry address, secret) for building your Workflows
 
-The `KogitoServerlessPlatform` is the Custom Resource used to control the behavior of the {operator_name}.
+The `SonataFlowPlatform` is the Custom Resource used to control the behavior of the {operator_name}.
 It defines the behavior of the operator when handling all {product_name} Custom Resources (Workflow and Build) in the given namespace.
 
-Since the operator is installed in global mode, you will need to specify a `KogitoServerlessPlatform` in each Namespace where you want to deploy Workflows.
-You can find a basic `KogitoServerlessPlatform` Custom Resource example in the link:https://github.com/kiegroup/kogito-serverless-operator/tree/{operator_version}/config/samples[`config/samples` folder] that you can simply apply to configure your operator.
+Since the operator is installed in global mode, you will need to specify a `SonataFlowPlatform` in each Namespace where you want to deploy Workflows.
 
-.Create a `KogitoServerlessPlatform`
-[source,bash,subs="attributes+"]
+[TIP]
+====
+If you have deployed a workflow for xref:cloud/operator/developing-workflows.adoc[development] you already have a `SonataFlowPlatform` custom resource. Just edit the one you have.
+====
+
+Following a very basic `SonataFlowPlatform` Custom Resource example to work on Kubernetes:
+
+.Example of a basic SonataFlowPlatform on Kubernetes
+[source,yaml,subs="attributes+"]
 ----
-kubectl apply -f https://raw.githubusercontent.com/kiegroup/kogito-serverless-operator/{operator_version}/config/samples/sw.kogito_v1alpha08_kogitoserverlessplatform.yaml -n kogito-workflows
+apiVersion: sonataflow.org/v1alpha08
+kind: SonataFlowPlatform
+metadata:
+  name: sonataflow-platform
+spec:
+  platform:
+    buildStrategyOptions:
+      KanikoBuildCacheEnabled: "true"
+    registry:
+      address: quay.io/kiegroup
+      secret: regcred
 ----
 
 [NOTE]
 ====
 In this Custom Resource, `spec.platform.registry.secret` is the name of the Secret you created just before.
```

Comment:
Here is better to remind the replacement of quay.io/kiegroup with the customer's registry described later

Reply:
I removed this note and added a description for each line.